src/hooks/useSession.js

```diff
@@ -0,0 +1,15 @@
+import { useSelector } from 'react-redux';
+import { createSelector } from 'reselect';
+
+const selectSession = createSelector(
```

Comment:
I'm not sure if we need to use `reselect` in this simple selector.
Using memoized selectors for such simple operation as accessing the props it doesn't worth it.

`reselect` will save the last `state.session` value and do a shallow comparison, also it will save the result value. In the end, this selector will be more "expensive" than the operation itself.

```suggestion
const selectSession = ({ session }) => ({
  authenticated: session.authenticated,
  checked: session.checked,
  user: session.user
})
```

Reply:
Thanks for the explanation. Addressed now.